src/Logic.hs

```diff
@@ -480,6 +482,30 @@ handlePullRequestEdited prId newTitle newBaseBranch state =
     -- Do nothing if the pull request is not present.
     Nothing -> pure state
 
+-- | If there is a push to a branch
+-- then update the integration status for the pullrequests
+-- that are being integrated and target the pushed branch.
```

Comment:
The structure of this sentence makes this quite hard to read. Do you mean something like:

```suggestion
-- | When we receive a push event, find all PRs 
-- that are in progress and have the same base 
-- branch. Reset their integration status to @Outdated@.
--
-- Avoid re-integrating due to pushing our own
-- SHAs of successfully integrated PRs.
```